daemon/src/maker_cfd.rs

```diff
@@ -618,8 +628,25 @@ where
 
     async fn handle(&mut self, FromTaker { taker_id, msg }: FromTaker) {
         match msg {
-            wire::TakerToMaker::TakeOrder { order_id, quantity } => {
-                if let Err(e) = self.handle_take_order(taker_id, order_id, quantity).await {
+            wire::TakerToMaker::TakeOrder {
+                order_id,
+                quantity,
+                leverage,
+            } => {
+                // Old clients do not send over leverage. Hence we default to Leverage::TWO which
+                // was the default before. We log a warning though.
+                let leverage = match leverage {
+                    None => {
+                        tracing::warn!(taker_id = %taker_id, "Received take request without leverage. Could be an outdated client but worth to double check the taker's version.");
```

Comment:
🙃 I think we can downgrade this to `debug`. If it is something that can happen (i.e. is expected to some degree) I don't think it should be a warning.

Reply:
You are right, it will happen particulary at the beginning when upgrading to a new version...
Happy to downgrade it but then I'm afraid we will just not see this message in the flood of other log messages.